src/app/shared/components/template/components/points-item/points-item.component.scss

```diff
@@ -1,5 +1,7 @@
 @use "/src/theme/mixins";
 
+$points-item-bg: var(--points-item-bg, var(--ion-color-primary-200));
```

Comment:
I wonder if this might be a good time to start adding consistency to how we refer to things like background. E.g. should it be `points-item-bg` or `points-item-background` (thinking about what is used elsewhere).

The benefit to this is we can a later stage offer a much simpler override system, where an author can specify at the component level something like `background: 'secondary-light'` and we can interpret as setting the corresponding variable `points-item-background: var(--color-secondary-light)` or similar

So the two things to keep in mind for this trying to keep component-specific prefixes and standard suffixes reasonably consistent where possible.

Doesn't have to all be done now, just something worth flagging